ui/src/store/resource.test.ts

```diff
@@ -683,7 +683,7 @@ describe('Store functions', () => {
         store.setURLParams('?category=Automation%2CBuild+Tools&catalog=tekton');
         store.parseUrl();
 
-        expect(store.filteredResources.length).toBe(1);
+        expect(store.filteredResources.length).toBe(7);
```

Comment:
The result will change I guess once we update the logic for filter

Reply:
yes